core/amber/src/main/scala/edu/uci/ics/amber/engine/common/promise/RPCHandlerInitializer.scala

```diff
@@ -0,0 +1,29 @@
+package edu.uci.ics.amber.engine.common.promise
+
+import com.twitter.util.{Future, Promise}
+import edu.uci.ics.amber.engine.common.ambertag.neo.VirtualIdentity.ActorVirtualIdentity
+import edu.uci.ics.amber.engine.common.promise.RPCServer.RPCCommand
+
+import scala.reflect.ClassTag
+
+class RPCHandlerInitializer(rpcClient: RPCClient, rpcServer: RPCServer) {
+
+  def registerHandler[B, C: ClassTag](handler: C => B)(implicit ev: C <:< RPCCommand[B]): Unit = {
```

Comment:
I see this has changed but there are no comments to explain implicit or the two sets of functions here. Also I dont see my comment anywhere.

Reply:
I'm handling the comments now